pymoto/modules/assembly.py

```diff
@@ -213,7 +213,7 @@ def _prepare(self, domain: DomainDefinition, *args,
 
         # Numerical integration
         siz = domain.element_size
-        w = np.prod(siz/2)
+        w = np.prod(siz[:domain.dim]/2)
```

Comment:
I recall this was a change also made in previous pull request; is this a duplication?

Reply:
Yes, that looks like it. The newest version of master needs to be merged into this branch

Reply:
I think I fixed this?